Sources/SwiftLanguageService/CodeCompletionSession.swift

```diff
@@ -704,6 +714,50 @@ class CodeCompletionSession {
 
     return Position(line: completionPos.line, utf16index: deletionStartUtf16Offset)
   }
+
+  /// Generate a snippet for control flow keywords like if, for, while, etc.
+  /// Returns the snippet text if the keyword is a control flow keyword and snippets are supported, otherwise nil.
+  private func keywordSnippet(for keyword: String) -> String? {
+    guard clientSupportsSnippets else { return nil }
+
+    func indentationUnitString() -> String {
+      // Convert the inferred Trivia (spaces/tabs) to a string we can embed into snippets.
+      if let trivia = indentationWidth {
```

Comment:
Nice, I forgot that we already have the inferred indentation in `CodeCompletionSession` 🤩